src/ast/recfun_decl_plugin.cpp

```diff
@@ -606,7 +606,7 @@ namespace recfun {
         : m_lhs(from.m_lhs),
           m_def(from.m_def),
           m_args(from.m_args) {}
-    case_expansion::case_expansion(case_expansion && from)
+    case_expansion::case_expansion(case_expansion && from) noexcept
```

Comment:
these copy/move constructors should just be deleted

Reply:
@nunoplopes Just this one? Or more?

Reply:
I only spotted this one, but there might be others. The ones that do reference counting can't go; everything else should go.

Reply:
Note that case-expansion is not on a hot path so seems overtuned to add a move constructor here in the first place.

Reply:
In the same file, there was a move constructor for `body_expansion` as well (which was already `noexcept`), so I removed that one as well.